stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/add/AddChannelViewModel.kt

```diff
@@ -0,0 +1,139 @@
+package io.getstream.chat.ui.sample.feature.channel.add
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.api.models.QueryUsersRequest
+import io.getstream.chat.android.client.channel.ChannelClient
+import io.getstream.chat.android.client.models.Filters
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.livedata.ChatDomain
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class AddChannelViewModel : ViewModel() {
```

Comment:
It should be in another module. shouldn't it?

Reply:
@ogkuzmin this one is in the right module. We are not gonna include AddChannelView in our SDK for now and I'm going to move all other related classes to the sample app in a separate PR